Create readme.md (#16)
TexusDev/Texus: examples/replit/readme.md

# Replit

> Replit allows users to write code and build apps and websites using a browser.The site also has various collaborative features, 

Reviewer: ```suggestion
Replit allows users to write code and build apps and websites using a browser. The site also has various collaborative features, 
```

including capability for real-time, multi-user editing with a live chat feed. It supports over 50 programming and markup languages, including Java, Python, and HTML, enabling users to build apps and websites. 

> The site is integrated with GitHub, a code-hosting platform, providing a way to import and run projects from GitHub.Users can also import projects from Glitch, which provides a similar service to Replit.

Reviewer: ```suggestion
The site is integrated with GitHub, a code-hosting platform, providing a way to import and run projects from GitHub Users can also import projects from Glitch, which provides a similar service to Replit.
```

# Installing

> Before installing Texus you are needed to uninstall forks such as `pycord, nextcord, etc.` or it will interfere with the library installation and usage

Reviewer: ```suggestion
Before installing Texus you need to uninstall forks such as `pycord, nextcord, etc.` or it will interfere with the library installation and usage
```

> Steps to install the latest version of texus :

Reviewer: ```suggestion
Steps to install the latest version of texus:
```

1. Head to console and type the following command `pip install texus`
2. Remove any type of discord.py fork or discord.py itself by doing `pip unsintall <package name>`
3. Make a file called `.replit` and inside the file you need to paste [this code](www.google.com)
4. If you are using nextcord you need to change everything from nextcord into discord in order to use texus.

> 5. You should be good to go after folowing the Steps if any questions feel free to join our support server.

Reviewer: ```suggestion
5. You should be good to go after folowing the Steps if any questions feel free to join our (support server)[https://discord.gg/54f7UWNsAJ]
```

> 5. You should be good to go after folowing the Steps if any questions feel free to join our [support server](https://discord.gg/54f7UWNsAJ)

Reviewer: ```suggestion
5. You should be good to go after following the Steps if any questions feel free to join our [support server](https://discord.gg/54f7UWNsAJ)
```
